### PR TITLE
Release 1.25.3

### PR DESCRIPTION
**Pull request by hakman**

### CODE REVIEW

File: docs/contributing/release-process.md [hunk 1]
```diff
@@ -52,13 +52,13 @@ In order to create the first beta release for a new minor version and a new rele
 
 ### Send Pull Request to propose a release
 
-See [1.22.0-beta.2 PR](https://github.com/kubernetes/kops/pull/12467) for an example.
+See [1.25.3-beta.2 PR](https://github.com/kubernetes/kops/pull/12467) for an example.
```
Comment (johngmyers) [inline docs/contributing/release-process.md:55]: I'm not sure updating this doc in the release commit makes sense, but if so the PR should change as well.

File: docs/contributing/release-process.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ git add . && git commit -m "Release ${VERSION}"
 This is the "release commit". Push and create a PR.
 
 ```
-gh pr create -f -l tide/merge-method-squash
+gh pr create -f -l tide/merge-method-squash -B release-1.25
```
Comment (johngmyers) [inline docs/contributing/release-process.md:76]: As a separate PR we should update the master-branch version of this document to specify -B, mentioning the value to use for a branch.

Comment (hakman) [reply docs/contributing/release-process.md]: Yeah, was a miss. Removed.

File: docs/contributing/release-process.md (hunk 1 above)
Comment (hakman) [reply docs/contributing/release-process.md]: Yes, was just notes. Removed.

**Review by johngmyers — APPROVED** [review]

**Pull request merged**